src/main/java/mate/academy/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import mate.academy.dao.CarDao;
+import mate.academy.db.Storage;
+import mate.academy.lib.Dao;
+import mate.academy.model.Car;
+import mate.academy.model.Driver;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+
+    @Override
+    public Car create(Car car) {
+        Storage.addCar(car);
+        return car;
+    }
+
+    @Override
+    public Car get(Long id) {
+        return Storage.cars.stream()
+                .filter(d -> d.getId().equals(id))
+                .findFirst()
+                .orElse(null);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return Storage.cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        IntStream.range(0, Storage.cars.size())
+                .filter(i -> Storage.cars.get(i).getId().equals(car.getId()))
+                .forEach(i -> Storage.cars.set(i, car));
+        return null;
```

Comment:
return null is bad practice
```suggestion
        return car;
```